README.md

```diff
@@ -128,6 +128,8 @@ The Service SIG meets every other week on [Wednesday at 9 AM PT](https://calenda
 
 Meeting notes are held in [this doc](https://docs.google.com/document/d/1dlcx3Mn3njG8lVmrgRFdGczTlyEN9IBO26cUwFQdlC8/edit?usp=sharing).
 
+Link to join the meeting: https://splunk.zoom.us/j/742947018.
```

Comment:
Please use cncf zoom meetings. They are recorded and published to YouTube

Reply:
https://github.com/open-telemetry/community/blob/master/docs/how-to-handle-public-calendar.md

Reply:
Can you please give me edit access to public calendar. The linked doc says I should have edit rights as a maintainer but I don't.

Reply:
@mtwo can you help with this?